traffic_portal/grunt/browserify2.js

```diff
@@ -89,7 +94,7 @@ module.exports = {
                         cwd: '<%= globalConfig.app %>/bower_components/',
                         src:
                             [
-                                'angular/angular.js',
+                                // 'angular/angular.js',
```

Comment:
you want to put this back and we can do this in another PR?

Reply:
Yeah, that wasn't meant to be in there.